src/aere_repl.erl

```diff
@@ -477,6 +483,11 @@ get_ready_chain(_) ->
     throw({repl_error, aere_msg:chain_not_ready()}).
 
 set_option(Option, Args, S = #repl_state{options = Opts}) ->
+    Locked = maps:get(locked_opts, Opts, []),
+    case lists:member(Option, [locked_opts|Locked]) of
```

Comment:
Because I don't want to let the user unlock a locked option (and give a meaningful message)

Reply:
I don't think this is really necessary since it's already included in the release version is being used for the docker image, so I have changed this back to false. If we found that this is causing any problems in the future, we can bring it back.